packages/python/readme_metrics/PayloadBuilder.py

```diff
@@ -175,8 +181,8 @@ def _build_response_payload(self, response: ResponseInfoWrapper) -> dict:
         for k, v in headers.items():
             hdr_items.append({"name": k, "value": v})
 
-        status_code = int(response.status.split(" ")[0])
-        status_text = response.status.replace(str(status_code) + " ", "")
+        status_code = int(str(response.status).split(" ")[0])
```

Comment:
Since you're doing `str(response.status)` twice, maybe we should assign that to a `status` and then use that instead.

Reply:
sgtm